docs/getting-started/backends-and-brokers/rabbitmq.rst

```diff
@@ -172,3 +172,62 @@ but rather use the :command:`rabbitmqctl` command:
     $ sudo rabbitmqctl stop
 
 When the server is running, you can continue reading `Setting up RabbitMQ`_.
+
+.. _using-quorum-queues:
+
+Using Quorum Queues
+===================
+
+.. warning::
+
+    Quorum Queues require disabling global QoS which means some features won't work as expected.
+    See `limitations`_ for details.
+
+Celery supports `Quorum Queues`_ by setting the ``x-queue-type`` header to ``quorum` like so:
+
+.. code-block:: python
+
+    from kombu import Queue
+
+    task_queues = [Queue('my-queue', queue_arguments={'x-queue-type': 'quorum'})]
+    broker_transport_options = {"confirm_publish": True}
+
+If you'd like to change the type of the default queue, set the :setting:`task_default_queue_type` setting to ``quorum``.
+
+Celery automatically detects if quorum queues are used using the :setting:`worker_detect_quorum_queues` setting.
+We recommend to keep the default behavior turned on.
+
+.. _`Quorum Queues`: https://www.rabbitmq.com/docs/quorum-queues
```

Comment:
Please also add a link to this article [Migrating from Mirrored Classic Queues to Quorum Queues](https://www.rabbitmq.com/blog/2023/03/02/quorum-queues-migration)

Reply:
I added the link.